client/src/components/manageProjects/utilities/validateEventForm.js

```diff
@@ -1,19 +1,24 @@
 import validator from 'validator';
-import { isAnySubstring } from './../../../utils/stringUtils.js';
-import { useProjectsStore } from '../../../store/projectStore.js';
+import { isWordInArrayInString } from './../../../utils/stringUtils.js';
 
-const validateEventForm = (vals) => {
+const validateEventForm = (vals, projectToEdit) => {
+  console.log(projectToEdit.name)
   let newErrors = {};
   Object.keys(vals).forEach((key) => {
     switch (key) {
       case 'name':
         // Required
         if (!vals[key]) {
           newErrors = { ...newErrors, name: 'Event name is required' };
-        } else if (isAnySubstring(['meeting', 'mtg'], vals[key])) {
+        } else if (
+          isWordInArrayInString(
+            ['meeting', 'mtg', projectToEdit.name.toLowerCase()],
```

Comment:
If using the above `projectToEdit = null` then you will also need to test this line to see if `projectToEdit` was suppied:
```js
['meeting', 'mtg', projectToEdit && projectToEdit.name.toLowerCase()]
```